### DISCUSSION TITLE
Temperatura Cronotermostato Moes/Tuya

### DISCUSSION THREAD

**Discussion by befaleg** [indomus/forum#989] [GENERALE - Discussioni generiche]
Ciao a tutti ho integrato in home assist un cronotermostato Moes tramite l'integrazione Tuya
Purtroppo quando creo la scheda climate in lovelace di home assistant mi da una temperatura sballata
avevo letto che era un difetto dei termostati cinesi e c'era un modo per correggere l'errore ti temperatura ma non trovo più i riferimenti.
Qualcuno mi può aiutare
Grazie

**Comment by Osky9** [comment]
Vai nelle impostazioni dell'integrazione e puoi sistemare. Il mio dava il doppio della temperatura e ho impostato che mi visualizzasse metà del valore

**Reply by befaleg** [reply]
ho controllato ma purtroppo nelle impostazioni dell'integrazione non trovo alcun modo di sistemarlo ,,,, il mio comunque rileva una temperatura minore devo moltiplicare tutto per 5

**Comment by Osky9** [comment]
Strano, io vado nel menù Integrazioni e nella "casella" Tuya dopo aver cliccato su CONFIGURA, mi fa selezionare il dispositivo da configurare e nella pagina successiva ho diversi valori impostabili tra cui il valore di divisore.

**Comment by vinny74man** [comment]
Il mio ha la voce offset che mi permette di aggiustare la temperatura letta, forse anche il tuo a questa voce
